components/eam/cime_config/config_compsets.xml

```diff
@@ -106,6 +106,16 @@
     <lname>2010_EAM%CHEMUCI-LINOZV3-MAM5_ELM%SPBC_MPASSI%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
   </compset>
 
+  <compset>
+    <alias>FSSP370</alias>
+    <lname>SSP370_eam_EAM%CMIP6_ELM%SPBC_MPASSI%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
+  </compset>
+
+  <compset>
+    <alias>FSSP585</alias>
+    <lname>SSP585_eam_EAM%CMIP6_ELM%SPBC_MPASSI%PRES_DOCN%DOM_MOSART_SGLC_SWAV</lname>
```

Comment:
What does the "_eam_" do in this longname?  I didn't think that was allowed in CIME.

Reply:
Hi Rob, it was used as the F for SSP585 compset when we were testing using the original alpha01 version. It is used to test the linoz-v2 configuration for future compset. Under this compset, we used several future scenario forcings as like done before. In the naming convention of correspondent files in use_cases of eam namelist_files, there was one with SSP585_cam5_CMIP6_bgc.xml, so I named it with eam to discriminate between the two.